doc/manual/15-system_requirements.en.md

```diff
@@ -10,7 +10,7 @@
     - python-dateutil
     - setuptools
 - PostgreSQL >= 10 (next version will require PostgreSQL >= 11)
-- rsync >= 3.0.4 (optional)
+- rsync >= 3.1.2 (optional)
```

Comment:
The PR looks good.

From the [changelog](https://download.samba.org/pub/rsync/NEWS#3.1.0) and PR description/commit message I can see we actually require `3.1.0` for `--ignore-missing-args` to be available.

I can see an internal comment about `3.1.2` being the oldest available in supported operating systems, so I guess that's why we are setting `3.1.2` here. But that seems more of a packaging characteristic than a requirement of Barman.

With that in mind I wonder if we should take some of the following actions:

* Set `3.1.0` as a requirement, and have `>= 3.1.0` in the package requirements; or
* Adjust the commit message/PR description to clarify why we are setting `3.1.2` in the markdown docs file instead of `3.1.0`, just so the history will make sense if one ever attempts to inspect the commit log

_I know this comment seems an overkill, but I felt I should raise it just in case_

Thoughts?

Reply:
think this makes sense. I'll push another commit to over-write the version dropping to 3.1.0